src/app/components/events/create-event/create-event.component.scss

```diff
@@ -0,0 +1,16 @@
+.app-form {
+  padding-top: 0.3rem;
+}
+
+.section-title {
+  box-sizing: border-box;
+  margin-top: 1rem;
+  margin-bottom: 1rem;
+
+  font-size: 1.4rem;
+  font-weight: bold;
+}
+
+.bets-container {
+  margin-bottom: 2rem;
```

Comment:
could be taken from styles.css instead of defined here. that way, you can use margin-bottom: 2rem; in every component without having to define it everywhere